src/WebInterface/Presentation/Http/View/game.html.twig

```diff
@@ -9,28 +9,42 @@
-                                        <path d="M6 21v-2a4 4 0 0 1 4 -4h4a4 4 0 0 1 4 4v2"/>
-                                    </svg>
-                                    Anonymous
-                                </div>
-                                <div class="col-6 col-xl-12">
-                                    <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"
-                                         fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
-                                         stroke-linejoin="round" class="icon icon-filled text-yellow">
-                                        <path stroke="none" d="M0 0h24v24H0z" fill="none"/>
-                                        <path d="M8 7a4 4 0 1 0 8 0a4 4 0 0 0 -8 0"/>
-                                        <path d="M6 21v-2a4 4 0 0 1 4 -4h4a4 4 0 0 1 4 4v2"/>
-                                    </svg>
-                                    Anonymous
-                                </div>
-                            </div>
+                            <p class="mb-1">
+                                <connect-four-player-status player-id="{{ app.user ? app.user.userIdentifier }}"
+                                                            color="red"
+                                                            color-player-id="{{ game.redPlayerId }}"
+                                                            current-player-id="{{ game.currentPlayerId }}"
+                                                            winning-player-id="{{ game.winningPlayerId }}"
+                                                            losing-player-id="{{ game.losingPlayerId }}"
+                                                            resigning-player-id="{{ game.resigningPlayerId }}"
+                                                            aborting-player-id="{{ game.abortingPlayerId }}"
+                                                            text-you="You"
+                                                            text-won="Won"
+                                                            text-lost="Lost"
+                                                            text-resigned="Resigned"
+                                                            text-aborted="Aborted">
+                                    <span class="status-dot status-red"></span>
+                                    <span data-username>Anonymous</span>
+                                </connect-four-player-status>
+                            </p>
+                            <p>
+                                <connect-four-player-status player-id="{{ app.user ? app.user.userIdentifier }}"
+                                                            color="yellow"
+                                                            color-player-id="{{ game.yellowPlayerId }}"
```

Comment:
Everything is duplicated from here. The element could be renamed to `connect-four-players` that handles both players, or there should be an include/macro for this.